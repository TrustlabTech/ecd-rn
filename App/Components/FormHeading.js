import React, { Component } from 'react'
import {
  View,
  Text,
  StyleSheet
} from 'react-native'
import { FontSizes } from '../GlobalStyles'

export default class FormHeading extends Component {

  render() {
    return (
      <View style={{marginTop: 5, marginBottom: 5, marginLeft: 25}}>
        <Text style={{fontWeight: 'bold', fontSize: FontSizes.h5}}>{this.props.text}</Text>
      </View>
    )
  }
}
