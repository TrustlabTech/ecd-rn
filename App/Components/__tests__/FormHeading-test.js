/**
 * Early Childhood Development App
 * @copyright 2016 Global Consent Ltd
 * Civvals, 50 Seymour Street, London, England, W1H 7JG
 * @author Werner Roets <werner@io.co.za>
 */

import 'react-native'
import React from 'react'
import FormHeading from '../FormHeading'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <FormHeading text="Test FormHeading"/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
