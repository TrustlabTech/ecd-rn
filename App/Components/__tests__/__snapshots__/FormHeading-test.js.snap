exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "marginBottom": 5,
      "marginLeft": 25,
      "marginTop": 5,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 22,
        "fontWeight": "bold",
      }
    }>
    Test FormHeading
  </Text>
</View>
`;
